From 9d52f886ffe556f9615f98a280d3a036a9ecf28e Mon Sep 17 00:00:00 2001
From: Kjell Ahlstedt <kjell.ahlstedt@bredband.net>
Date: Sun, 5 Jul 2015 15:41:11 +0200
Subject: 	More fixes to suit newish glibmm.

	* gtk/src/entrycompletion.hg: Add a missing double quote in
	_WRAP_PROPERTY("text_column, int), to avoid a warning from gmmproc 2.41.0
	or newer.
	* gtk/src/printunixdialog.hg: get_selected_printer(): Ref-counting correction.
	Fix a .->, typo to make the refreturn option actually be used. Necessary
	with gmmproc 2.29.13 or newer. Discussed in bug #657751, comment 5.

diff --git a/ChangeLog b/ChangeLog
index 1d6d4d0..c61ffa3 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,14 @@
+2015-07-05  Kjell Ahlstedt  <kjell.ahlstedt@bredband.net>
+
+	More fixes to suit newish glibmm.
+
+	* gtk/src/entrycompletion.hg: Add a missing double quote in
+	_WRAP_PROPERTY("text_column, int), to avoid a warning from gmmproc 2.41.0
+	or newer.
+	* gtk/src/printunixdialog.hg: get_selected_printer(): Ref-counting correction.
+	Fix a .->, typo to make the refreturn option actually be used. Necessary
+	with gmmproc 2.29.13 or newer. Discussed in bug #657751, comment 5.
+
 2015-07-02  Kjell Ahlstedt  <kjell.ahlstedt@bredband.net>
 
 	Add #includes needed with glibmm newer than 2.31.0.
diff --git a/gtk/src/entrycompletion.hg b/gtk/src/entrycompletion.hg
index 3a32e4f..70c5cc7 100644
--- a/gtk/src/entrycompletion.hg
+++ b/gtk/src/entrycompletion.hg
@@ -218,7 +218,7 @@ dnl
 
   _WRAP_PROPERTY("model", Glib::RefPtr<Gtk::TreeModel>)
   _WRAP_PROPERTY("minimum_key_length", int)
-  _WRAP_PROPERTY("text_column, int)
+  _WRAP_PROPERTY("text_column", int)
   _WRAP_PROPERTY("inline_completion", bool)
   _WRAP_PROPERTY("popup_completion", bool)
   _WRAP_PROPERTY("popup_set_width", bool)
diff --git a/gtk/src/printunixdialog.hg b/gtk/src/printunixdialog.hg
index 87de89f..c05e5c3 100644
--- a/gtk/src/printunixdialog.hg
+++ b/gtk/src/printunixdialog.hg
@@ -71,7 +71,7 @@ public:
   #m4 _CONVERSION(`GtkPrinter*',`Glib::RefPtr<const Printer>',`Glib::wrap($3)')
 
   _WRAP_METHOD(Glib::RefPtr<Printer> get_selected_printer(), gtk_print_unix_dialog_get_selected_printer, refreturn)
-  _WRAP_METHOD(Glib::RefPtr<const Printer> get_selected_printer() const, gtk_print_unix_dialog_get_selected_printer, refreturn. constversion)
+  _WRAP_METHOD(Glib::RefPtr<const Printer> get_selected_printer() const, gtk_print_unix_dialog_get_selected_printer, refreturn, constversion)
 
   void add_custom_tab(const Widget& child, const Glib::ustring& tab_label);
   _WRAP_METHOD(void add_custom_tab(const Widget& child, const Widget& tab_label), gtk_print_unix_dialog_add_custom_tab)
-- 
cgit v0.10.2

